include/aspect/material_model/interface.h

```diff
@@ -222,6 +222,25 @@ namespace aspect
       std::vector<double> viscosities;
 
       /**
+       * Constitutive tensors at the given positions. This variable
+       * can be used to implement exotic rheologies such as anisotropic
+       * viscosity.
+       *
+       * @note The strain rate term in equation (1) of the manual will be
+       * multiplied by this tensor *and* the viscosity scalar (\\eta). If no
+       * value is assigned here the identity tensor will be used by default.
+       *
+       * @note because the use of this variable generalizes the constitutive
+       * relation between strain rate and divergence of stress, it is ambiguous
+       * how it should interact with the div(u) term in equation (1) of the
+       * manual. It is therefore only multiplied by the strain rate term in
+       * the governing equations and left up to the user to proceed with
+       * caution when implementing compressible material models which use the
+       * constitutive tensor.
+       */
+      std::vector<SymmetricTensor<4,dim> > consitutive_tensors;
```

Comment:
there's a misspelling in the name of the variable that you probably want to fix throughout.